js/windowMgr.js

```diff
@@ -174,6 +175,28 @@ function doCreateMainWindow(initialUrl, initialBounds) {
 
     mainWindow.on('closed', destroyAllWindows);
 
+    // Manage File Downloads
+    mainWindow.webContents.session.on('will-download', (event, item, webContents) => {
```

Comment:
good question about 3rd party... currently don't believe we have any support or need there.  but i'll touch base with app group.  for now we can skip as i know we don't support in current wrapper.